go/libkb/wot.go

```diff
@@ -198,48 +212,35 @@ func fetchWot(mctx MetaContext, username *string) (res []serverWotVouch, err err
 	return response.Vouches, nil
 }
 
-func FetchMyWot(mctx MetaContext) (res []keybase1.WotVouch, err error) {
-	defer mctx.Trace("FetchMyWot", func() error { return err })()
-	serverVouches, err := fetchWot(mctx, nil)
+// FetchUserWot gets vouches written for vouchee (if specified) by voucher (if
+// specified).
+func FetchUserWot(mctx MetaContext, vouchee *string, voucher *string) (res []keybase1.WotVouch, err error) {
```

Comment:
i think passing in a typed arg is a better approach here for 2 reasons:
1. it's _much_ more future proof. i strongly suspect there will be additional usecases for fetching different slices of WotVouch. with an arg, it's as simple as adding the parameters, adding the checks, and adding the behavior. the way this is written, the person coming in to change it will have to decide whether to add an argument to the method signature (which means changing every call to the method) or adding a new method with a bunch more code and a bunch of duplicated behavior. 
2. it signals to someone reading this code that this is the one entry point for fetching web of trust vouches from the server. it makes reading and tracing code easier generally when things like this are easy to identify.

Reply:
1. can you give me an idea of what additional args we want to add to Fetch?
2. i think it's pretty clear this is the one entry point given that it's one of the only public methods in the file, and how other methods call it :)

Reply:
off the top of my head, i could see wanting to query:
* what did i write that hasn't been responded to / has been rejected / has suggested edits
* who are some people i mutually follow that i haven't written about yet
* who are some people i've written about that haven't written about me